spec.html

```diff
@@ -15855,6 +15856,8 @@ <h1>GetSuperConstructor ( )</h1>
           1. Assert: _envRec_ is a function Environment Record.
           1. Let _activeFunction_ be _envRec_.[[FunctionObject]].
           1. Assert: _activeFunction_ is an ECMAScript function object.
+          1. If _activeFunction_.[[ConstructorKind]] is *"base"*, return *null*.
+          1. Assert: _activeFunction_.[[ConstructorKind]] is *"derived"*.
```

Comment:
This will now fail, because `[[ConstructorKind]]` is a spec enum and not a runtime string value.
```suggestion
```